server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EmailGetMethodContract.scala

```diff
@@ -3051,7 +3051,7 @@ trait EmailGetMethodContract {
          |    "error",
          |    {
          |      "type": "unknownMethod",
-         |      "description": "Missing capability(ies): urn:ietf:params:jmap:core, urn:ietf:params:jmap:mail"
```

Comment:
this commit must be reverted, see https://github.com/iNPUTmice/lttrs-android/issues/55 and https://mailarchive.ietf.org/arch/msg/jmap/wQaw_ivbvFtOrEPbkkOkDp3OGrU/

Reply:
:+1: